internal/environment/command_create.go

```diff
@@ -65,5 +65,5 @@ func (c *command) create(cmd *cobra.Command, args []string) error {
 
 func (c *command) addStreamGovernancePackageFlag(cmd *cobra.Command, defaultValue string) {
 	values := utils.ArrayToCommaDelimitedString([]string{"essentials", "advanced"}, "or")
-	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf("Specify the Stream Governance package as %s.", values))
+	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf(`Specify the Stream Governance package as %s. Downgrading the package from "advanced" to "essentials" is not allowed once the schema registry cluster is provisioned.`, values))
```

Comment:
```suggestion
	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf(`Specify the Stream Governance package as %s. Downgrading the package from "advanced" to "essentials" is not allowed once the Schema Registry cluster is provisioned.`, values))
```

Reply:
So if a user runs `confluent environment create test --governance-package advanced` followed by `confluent environment update --governance-package essentials` fast enough, it'll work, as long as the Schema Registry cluster isn't provisioned?

Reply:
SR cluster doesn't get provisioned until after Kafka cluster is created, so they would just need to run `confluent environment update test --governance-package essentials` before running `confluent kafka cluster create cluster_0`